### PR TITLE
[docs] POST /_aliases remove_index action only works on concrete indices

### PR DESCRIPTION
**Pull request by rudolf**
```
$ curl -X PUT "elastic:changeme@localhost:9200/.kibana_1"
{"acknowledged":true,"shards_acknowledged":true,"index":".kibana_1"}
$ curl -X PUT "elastic:changeme@localhost:9200/.kibana_1/_alias/.kibana"
{"acknowledged":true}
$ curl -X POST "elastic:changeme@localhost:9200/_aliases" -H 'Content-Type: application/json' -d'
{
  "actions" : [
    { "remove_index": { "indices": ".kibana" } }
  ]
}
'
{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"The provided expression [.kibana] matches an alias, specify the corresponding concrete indices instead."}],"type":"illegal_argument_exception","reason":"The provided expression [.kibana] matches an alias, specify the corresponding concrete indices instead."},"status":400}
```

### CODE REVIEW

File: docs/reference/indices/aliases.asciidoc [hunk 1]
```diff
@@ -62,8 +62,7 @@ Adds an alias to an index.
 Removes an alias from an index.
 
 `remove_index`::
-Deletes an index or index alias,
-like the <<indices-delete-index,delete index API>>.
+Deletes a concrete index. Similar to the <<indices-delete-index, delete index API>>, but if the provided index is an alias, the action will fail.
```
Comment (jrodewig) [inline docs/reference/indices/aliases.asciidoc:65]:
```suggestion
Deletes a concrete index, similar to the <<indices-delete-index, delete index
API>>. Attempts to remove an index alias will fail.
```

**Review by jrodewig — APPROVED** [review]
LGTM. I'm going to incorporate a minor edit.

**Comment by rudolf** [comment]
@jrodewig I missed that the delete API has the same behaviour so +1 for your edit. Maybe we should update my second edit as well:

> `remove_index` is just like <<indices-delete-index>> but will only remove a concrete index.

to become:

> `remove_index` is just like <<indices-delete-index>> and will only remove a concrete index.

**Comment by jrodewig** [comment]
Thanks for pointing that out @rudolf. I'll handle that as part of a follow-up PR or commit.

**Comment by jrodewig** [comment]
Updated the callout text with c7a11b5719d and backported to the appropriate branches.

**Pull request merged**